crates/orchestrator/src/workers/snos.rs

```diff
@@ -8,7 +14,35 @@ impl Worker for SnosWorker {
     /// 1. Fetch the latest completed block from the Starknet chain
     /// 2. Fetch the last block that had a SNOS job run.
     /// 3. Create SNOS run jobs for all the remaining blocks
+    // TEST : added config temporarily to test
     async fn run_worker(&self) {
-        todo!()
+        let config = config().await;
+        let provider = config.starknet_client();
+        let latest_block_number = provider.block_number().await.expect("Failed to fetch block number from rpc");
```

Comment:
should we make workers return a Result and throw an error wherever we do `unwrap` or `expect`

Reply:
that should be the ideal case but ig error is handled in `create_job` function.

Reply:
but we're doing unwraps and expects in worker, this will crash the orchestrator

Reply:
+1 
In most cases it's better (imho) to propagate the error as far as possible and let the caller decide what to do with it rather than panic

Reply:
added error throwing in workers